hooks/secondary_pre_publish_tk-houdini.py

```diff
@@ -97,3 +99,52 @@ def execute(self, tasks, work_template, progress_cb, user_data, **kwargs):
             progress_cb(100)
 
         return results
+
+    def __validate_item_for_alembic_cache_publish(self, item):
+        """
+        Validate that the item is valid to be exported to an alembic cache.
+        
+        :param item:    The item to validate
+        :returns:       A list of any errors found during validation that should be reported
+                        to the artist
+        """
+    
+        # validate the exported alembic cache and add any errors found here
+        errors = []
+
+        node = item["other_params"]["node"]
+        path = item["other_params"]["path"]
+        if not os.path.exists(path):
+            errors.append("No cache exists to be published!")
```

Comment:
This shows up in the "warnings" in the publisher when you've selected a node to publish but nothing is on disk. I was thinking the path might get cluttered in there and it shows the path to the node in houdini already I think. I updated the string to say "No alembic cache has been written to disk for this node.". Seem reasonable?